Code/Engine/RendererVulkan/Device/Implementation/DeviceVulkan.cpp

```diff
@@ -1133,6 +1129,11 @@ ezGALBuffer* ezGALDeviceVulkan::CreateBufferPlatform(
 void ezGALDeviceVulkan::DestroyBufferPlatform(ezGALBuffer* pBuffer)
 {
   ezGALBufferVulkan* pVulkanBuffer = static_cast<ezGALBufferVulkan*>(pBuffer);
+  for (ezInt32 i = (ezInt32)m_PendingBufferCopies.GetCount() - 1; i >= 0; --i)
```

Comment:
Since you do a swap with last anyways you don't need to iterate backwards I guess? Swap with last should be fine since the order doesn't really matter, we don't allow overlapping updates.

Reply:
Fixed! Not sure why I used a backwards loop. As you said, order is not important and I was already not maintaining the order anyways.